README-multiple-instance.MD

```diff
@@ -0,0 +1,72 @@
+Copyright (c) 2016 Intel Corporation 
```

Comment:
Convention is to have extension for README document in lower case as, 'README-multiple-instance.md'